src/app/cats/cats-data.service.ts

```diff
@@ -13,36 +13,60 @@ export class CatsDataService {
       name: 'Puszek',
       ageMonths: 8,
       breed: 'maine coon',
-      picUrl: 'puszek1.jpg'
+      picUrl: ['puszek1.jpg','puszek2.jpg','puszek3.jpg'],
+      vacinated : false,
+      funFact: "Głośno mruczy",
+      state: "wolny",
+      adoption : false
     },
     {
       name: 'Mruczek',
       ageMonths: 36,
       breed: 'brytyjczyk',
-      picUrl: 'mruczek1.jpg'
+      picUrl: ['mruczek1.jpg','mruczek2.jpg','mruczek3.jpg'],
+      vacinated : true,
+      funFact: "Wspina się na zasłony",
+      state: "wolny",
+      adoption: false,
     },
     {
       name: 'Kiciek',
       ageMonths: 52,
       breed: 'perski',
-      picUrl: 'kiciek.jpg'
+      picUrl: ['kiciek1.jpg', 'kiciek2.jpg','kiciek3.jpg'],
```

Comment:
wsadzenie dwa razy tego samego obrazka, w momencie gdy mamy przewijać obrazki, to raczej słabe date testowe.